packages/normalize/css/index.css

```diff
@@ -74,6 +81,10 @@ li {
   }
 }
 
+em {
```

Comment:
This was one of my more lighter-touch solution attempts.  It doesn't work by itself. But I left it because it makes sense for a normalize. But, on second thought, this should be browser default anyway, right?